### DISCUSSION TITLE
canary - parametrizace

### DISCUSSION THREAD

**Discussion by onhru** [canarytrace/documentation#205] [Synthetic]
Hezky den,

prosim o moznost pridani parametru:
- pro spousteni canarytrace v rezimu SMOKE umoznit zadat volitelne cookie. Potrebuji pro GA mit moznost odfiltrovat tyto smoke testy, aby busines mel presne informace o navstevnosti na danych strankach. (jde samozrejme o produkcni testy)
- pro spusteni canarytrace v jakemkoliv rezimu, mit moznost nastavit volitelny timeout mezi jednotlivymi testy, tak aby nedochazelo k posilani emergency zprav o tom ze canary nefunguje (tento email se posila pokud neni canary spusteno v predpokladanych intervalech)

S pranim hezkeho dne
Ondrej Hrusa - Direct-auto

**Comment by rdpanek** [comment]
Dobrý den @onhru pro bod 2 stáhněte prosím nový release Listeneru https://v2doc.canarytrace.com/docs/listener/releases#148

**Reply by rdpanek** [reply]
@onhru Je pro Vas pripraven novy docker image s Canarytrace, ktery umoznuje v rezimu Smoke nastavit cookies `docker pull quay.io/canarytrace/canarytrace-pub:4.24.0-pro-20231220130207-62` Více informací naleznete v https://github.com/canarytrace/documentation/issues/208
